### DISCUSSION TITLE
Generate a radar point cloud dataset with labelled data

### DISCUSSION THREAD

**Discussion by ecramo** [carla-simulator/carla#5688] [Q&A]
I need your help, because I don't know if in Carla it is possible to label the RADAR data when measuring. This question arises because if you know the object in the simulation and recognise where they are. I understand that it might be possible to cross-reference or combine this information, but how?, has it been tested?, do you actually know the location of the other vehicles that are being simulated in autopilot? 

thanks in advance

**Comment by kvnptl** [comment]
Any update on this?

**Reply by ecramo** [reply]
Any update..
